### DISCUSSION TITLE
ERROR FileEntryRepository.java:192 : Error while fetching files from SVN for admin 에러 질문드립니다.

### DISCUSSION THREAD

**Discussion by hyunzxn** [naver/ngrinder#968] [Q&A]
사용환경: 로컬
사용 장비 및 OS: Apple M1 Pro / Monterey
Java: 17.0.5

nGrinder 최신버전(3.5.8) 다운로드 받고 컨트롤러 실행, 에이전트 실행까지는 완료된 상태입니다.

script를 한 번 만들어보려고 컨트롤러에서 script 항목을 클릭하면 컨트롤러 쪽에 제목과 같은 에러 로그가 찍히면서 아무런 동작도 하고 있지 않은 상황입니다.

에이전트 쪽에서도 agent controller: Waiting for agent controller server signal 이 로그가 마지막으로 찍혀 있습니다.

혹시 어떤 부분이 문제가 되는지 알 수 있을까요? 필요한 정보 더 말씀해주시면 로그나 추가적인 정보 댓글로 추가하겠습니다.

**Comment by hyunzxn** [comment]
nGrinder HOME을 완전히 삭제했다가 다시 실행을 하고 나니 해결이 되었습니다.

**Reply by LeeJyeon** [reply]
nGrinder HOME 이라는 경로가 어딘지 알 수 있을까요?

**Reply by hyunzxn** [reply]
cd/users/{사용자명}/.ngrinder 이 경로 입니다.

**Reply by LeeJyeon** [reply]
감사합니다!

**Reply by small-j** [reply]
혹시 에러 로그를 통해서 문제 해결 방안을 찾으셨나요?
저도 같은 문제를 겪고 있는데, 문제 해결 방안이 감도 오지 않아서 어떻게 찾으셨는지 알게 되면 도움이 많이 될 것 같아 질문드립니다.
답변주시면 감사드리겠습니다!

**Reply by LeeJyeon** [reply]
@small-j 
/Users/{사용자명}/.ngrinder 폴더 통째로 삭제하고 다시 진행하면 됐던걸로 기억합니다
